lib/svgo/js2svg.js

```diff
@@ -118,6 +131,15 @@ JS2SVG.prototype.convert = function (data) {
 
   this.indentLevel--;
 
+  if (
+    this.config.finalNewline &&
+    this.indentLevel === 0 &&
+    svg.length > 0 &&
+    svg[svg.length - 1] !== '\n'
```

Comment:
Is there a case where resulting svg may contain final newline?

Reply:
Yes, when `pretty` is true.